mgmt/rest/metric.go

```diff
@@ -40,6 +41,7 @@ func (s *Server) getMetrics(w http.ResponseWriter, r *http.Request, _ httprouter
 	// perform a query
 	q := r.URL.Query()
 	v := q.Get("ver")
+	apiVersion := r.URL.Path[1:3]
```

Comment:
Maybe add a comment to explain it'll catch "v1", "v2". 

Do we want to handle an error if something is different than "v1" or "v2?